### ISSUE TITLE
What will be the content of this website?

### ISSUE THREAD

**Issue by qrilka** [reflex-frp/reflexfrp.org#1]
For sure links are to be there but I guess there should be something else?
Probably reflex examples?
Probably some docs like reflex platform README?

**Comment by ryantrinkle** [comment]
There's no official plan yet, but this would be a good place to hash that kind of stuff out!

I, for one, would love to see some documentation with interactive "marble diagrams" like some other reactive sites use.

**Comment by dfordivam** [comment]
Has there been any progress on this... any drafts or documents lying around?
else I will work on this from scratch and upload some drafts...
Reflex is in big need of an introduction and documentation...

**Comment by ryantrinkle** [comment]
@dfordivam Nope, we haven't had much time to apply to this.  It'd be great if you took the lead!

**Comment by dfordivam** [comment]
Some ideas...

This website need to have the following
1. Main page - Introducing reflex, highlighting its strengths, hsnippet examples, etc - Basically marketing stuff
  * This is intended for newbie.

2. Tutorial - May be start with content from reflex-platform
  * This is intended for newbie.

3. Examples - Content from reflex-examples, along with compiled code to view what is the outcome.
  * Need to provide examples not just for newbie, but for real world apps.
  * reflex-dom-contrib has some good content which can be used here.

4. Documentation - Need to gather data and create a nice user guide / documentation
  * This is intended for devs who are using Reflex.
  * It has to be comprehensive and maintained according to the stable release.

5. FAQs, Support, etc

**Comment by dfordivam** [comment]
I would like to start with documentation first. I will put up the draft in markdown for review.

**Comment by ryantrinkle** [comment]
@dfordivam That sounds like a fantastic start.  Please keep me up to date on how it goes!
